ui/src/actions/general.js

```diff
@@ -82,9 +82,12 @@ function notify(error) {
   console.error('REQUEST FAILED', error); // eslint-disable-line no-console
 }
 
-export function getInitialState(userInControl) {
-  return (dispatch) => {
+export function getInitialState() {
+  return (dispatch, getState) => {
     const state = {};
+    const { login } = getState();
+
+    dispatch(applicationFetched(false));
```

Comment:
I don't think this is needed, since this is the default value in the store on initialisation, and `getInitialState` is only dispatched once.